compiler/rustc_mir/src/borrow_check/diagnostics/conflict_errors.rs

```diff
@@ -260,18 +277,20 @@ impl<'cx, 'tcx> MirBorrowckCtxt<'cx, 'tcx> {
                 }
 
                 if let UseSpans::PatUse(span) = move_spans {
-                    err.span_suggestion_verbose(
-                        span.shrink_to_lo(),
-                        &format!(
-                            "borrow this field in the pattern to avoid moving {}",
-                            self.describe_place(moved_place.as_ref())
-                                .map(|n| format!("`{}`", n))
-                                .unwrap_or_else(|| "the value".to_string())
-                        ),
-                        "ref ".to_string(),
-                        Applicability::MachineApplicable,
-                    );
-                    in_pattern = true;
+                    if maybe_reinitialized_locations.is_empty() {
```

Comment:
You can merge these two lines with something like the following to avoid one level of indentation:

```rust
if let (UseSpans::PatUse(span), []) = (move_spans, &maybe_reinitialized_locations[..]) {
```

Reply:
Done